op-node/rollup/derive/params.go

```diff
@@ -30,14 +30,22 @@ const DerivationVersion1 = plasma.TxDataVersion1
 const MaxSpanBatchSize = MaxRLPBytesPerChannel
 
 // MaxChannelBankSize is the amount of memory space, in number of bytes,
-// till the bank is pruned by removing channels,
-// starting with the oldest channel.
+// till the bank is pruned by removing channels, starting with the oldest channel.
 const MaxChannelBankSize = 100_000_000
 
+// MaxChannelBankSizeV1 is the amount of memory space, in number of bytes,
+// till the bank is pruned by removing channels, starting with the oldest channel.
+// This limit is applied after Fjord
+const MaxChannelBankSizeV1 = 1_000_000_000
+
 // MaxRLPBytesPerChannel is the maximum amount of bytes that will be read from
 // a channel. This limit is set when decoding the RLP.
 const MaxRLPBytesPerChannel = 10_000_000
 
+// MaxRLPBytesPerChannelV1 is the maximum amount of bytes that will be read from
+// a channel after Fjord. This limit is set when decoding the RLP.
+const MaxRLPBytesPerChannelV1 = 100_000_000
```

Comment:
Maybe we should remove these constants from the public API of the `derive` package to avoid accidentally referencing the old constant. We would instead force ppl to get the protocol parameters from a rollup config (see other comment), avoiding using a go `const` that isn't a const across forks any more.

We could then also rename the private constants with suffixes `V0` and `V1` or even `...Bedrock` and `...Fjord`.